pennylane_lightning/core/src/simulators/lightning_kokkos/catalyst/LightningKokkosObsManager.hpp

```diff
@@ -0,0 +1,210 @@
+// Copyright 2023-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I wonder whether we should keep these dates if the file is moved between repositories.

Reply:
Good point! I argue that this was and is Xanadu copyright, so the dates should start when first created.